src/frontend.md

```diff
@@ -421,6 +421,58 @@ export function fakeTicket() {}
    // no additional network request needed, use the first one
    ```
 
+4. Add query filters and path params to query keys.
```

Comment:
wouldn't you like to use the same words as here?
https://github.com/OsomePteLtd/principles/pull/254#discussion_r1267924640

Reply:
To put it more clear, appended with this words.